zmq.cpp

```diff
@@ -37,75 +39,95 @@ void LoadBalancer::run() {
         }
         // handle worker
         if (items[0].revents & ZMQ_POLLIN) {
+
             // the first frame is the identifier of the worker: we add it to the available worker
             avalailable_worker.push(z_recv(workers));
```

Comment:
I know this is not part of your PR but since you're there :smile: 
```suggestion
            available_worker.push(z_recv(workers));
```